pombola/south_africa/templates/core/organisation_party.html

```diff
@@ -60,11 +61,17 @@
                   {% endmaybehidden %}
 
                   {% for position in positions %}
-                      {{ position.title }}
-                      {% if position.place %}
+                      {% should_display_position position.organisation position.title as display_position %}
+                      {% if display_position %}
+                        {{ position.title }}
+                      {% endif %}
+                      {% should_display_place position.organisation as display_place %}
+                      {% if display_place and position.place %}
                           (<a href="{{ position.place.get_absolute_url }}">{{ position.place.name }}</a>)
                       {% endif %}
-                      <br/>
+                      {% if display_position or display_place and position.place %}
```

Comment:
I'd be tempted to put a comment above (since I had to look this up) quoting https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#boolean-operators "Use of both and and or clauses within the same tag is allowed, with and having higher precedence than or e.g.:" and explaining that parentheses for precedence aren't allowed.

Or I dunno. Then I suppose you should add the same to the other file - maybe it's distracting noise. Up to you, feel free to ignore this :)